ingesters/s3Ingster/s3.conf

```diff
@@ -0,0 +1,27 @@
+[Global]
+Ingest-Secret = "IngestSecrets"
+Connection-Timeout = 0
+Insecure-Skip-TLS-Verify=false
+#Cleartext-Backend-Target=127.0.0.1:4023 #example of adding a cleartext connection
+#Cleartext-Backend-Target=127.1.0.1:4023 #example of adding another cleartext connection
+#Encrypted-Backend-Target=127.1.1.1:4024 #example of adding an encrypted connection
+Pipe-Backend-Target=/opt/gravwell/comms/pipe #a named pipe connection, this should be used when ingester is on the same machine as a backend
+#Ingest-Cache-Path=/opt/gravwell/cache/s3.cache #adding an ingest cache for local storage when uplinks fail
+#Max-Ingest-Cache=1024 #Number of MB to store, localcache will only store 1GB before stopping.  This is a safety net
+Log-Level=INFO
+Log-File=/opt/gravwell/log/sq3.log
+
+# A Queue pulls from a specific SQS queue with a given AKID and Secret. See
```

Comment:
Comments and structure indicate this is an SQS config. Is it just the comments that are wrong here or is this the wrong config?

Reply:
resolved